### DISCUSSION TITLE
How to avoid the impact of logs when doing performance testing in libraries that use tracing?

### DISCUSSION THREAD

**Discussion by jiesia** [tokio-rs/tracing#2595] [Q&A] [answered]
How to avoid the impact of logs when doing performance testing in libraries that use tracing?

**Comment by hds** [comment, marked as answer]
You can use compile time filters to turn all tracing off: https://docs.rs/tracing/latest/tracing/level_filters/index.html#compile-time-filters

You want the `max_level_off` filter feature (you specify this in Cargo.toml, see the docs at the link for an example).
